Abs.lua

```diff
@@ -1,3 +1,5 @@
+local THNN = require('nn.THNN')
```

Comment:
wouldn't it be better to include `THNN` in `nn` namespace, like `nn.THNN` or `nn._THNN` ? This would avoid having to require 'nn.THNN' at every lua file.